skyfield/constants.py

```diff
@@ -22,12 +22,15 @@
 ANGVEL = 7.2921150e-5
 ERAD = 6378136.6
 IERS_2010_INVERSE_EARTH_FLATTENING = 298.25642
+IERS_2010_EARTH_FLATTENING = 1 / IERS_2010_INVERSE_EARTH_FLATTENING
```

Comment:
What was the motivation for moving this out of `earthlib.py` — do you have plans to use it somewhere else as well, maybe in the SGP4 code?